ministryofjustice/analytics-platform-control-panel-frontend
# Allow apps/create repo typeahead to show all repos

Allow apps/create repo typeahead to show all repos from the two github organisations

## What

The repo typeahead when creating apps didn't show all repos. This is because we weren't paging the API

## How to review

1. go to `/apps/new`
2. type stuff in the typeahead, note that there are more results.
3. View source of the page and see that there are ~688 repos for the ministryofjustice organisation in the DOM
4. if you have any private repos in the mojanalytics org you should see them too

https://trello.com/c/8guRQftR/1095-repo-typeahead-in-create-webapp-page-is-not-fetching-all-repos-in-the-org
